alternator/executor.cc

```diff
@@ -883,7 +899,13 @@ future<executor::request_return_type> executor::list_tags_of_resource(client_sta
     return make_ready_future<executor::request_return_type>(make_jsonable(std::move(ret)));
 }
 
-static void verify_billing_mode(const rjson::value& request) {
+struct billing_mode_type {
+    bool provisioned = false;
+    int64_t rcu;
```

Comment:
I don't think we need to support more than 4 billion reads or writes per second, but maybe we should be prepared for the distant future, so ok :-)

Reply:
changed to int